ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml

```diff
@@ -60,6 +60,20 @@ tests:
     - chain: ipi-aws-pre
     - chain: workers-scale
     workflow: openshift-qe-router-perf
+- as: perfscale-aws-ingress-perf-periodic
+  cron: 0 12 * * 5
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      WORKER_REPLICA_COUNT: "20"
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
+    - chain: create-infra-move-ingress-monitoring-registry
+    - chain: workers-scale
+    workflow: openshift-qe-ingress-perf
```

Comment:
I am tackling this self-managed AWS job via this PR already:
https://github.com/openshift/release/pull/41725 

which is based on airflow DAG: 
https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/dags/openshift_nightlies/config/install/aws/ovn-dp-v2.json